demosplan/DemosPlanCoreBundle/Command/Addon/AddonUninstallCommand.php

```diff
@@ -56,19 +59,32 @@ protected function configure(): void
             InputArgument::OPTIONAL,
             'Name of the addon to uninstall. May be omitted to receive a list of installed addons.',
         );
+        $this->addOption('all', 'a', InputOption::VALUE_NONE, 'Uninstall all Addons');
     }
 
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $output = new SymfonyStyle($input, $output);
         $addonsInfos = $this->registry->getAddonInfos();
+        $all = $input->getOption('all');
 
         if (empty($addonsInfos)) {
             $output->info('No addons installed, nothing to uninstall');
 
             return self::SUCCESS;
         }
 
+        // Handle the --all option
+        if ($all) {
+            foreach ($addonsInfos as $addonInfo) {
+                $this->uninstallAddon($addonInfo, $output);
+            }
+            $this->clearCacheFolder($output);
```

Comment:
it will be better to check if environment === 'dev' here otherwise it's not a good idea to remove cache in prod environment this way